minard/views.py

```diff
@@ -618,5 +625,56 @@ def pca_run_detail(run_number):
     
     return render_template('pca_run_detail.html',
                             run_number=run_number)      
+
+@app.route('/hldq')
+def hldq():
+    #Load the run numbers
+    return 
+
+@app.route('/calibdq')
+def calibdq():
+        return render_template('calibdq.html')
    
+@app.route('/calibdq_tellie')
+def calibdq_tellie():
+    run_dict = {}
+    run_numbers = tools.import_TELLIE_runnumbers()
+    for num in run_numbers:
+            run_num, check_params, runInformation =  tools.import_TELLIEDQ_ratdb(num)
+            #If we cant find DQ info skip
+            if check_params == -1 or runInformation == -1:
+                continue
+            run_dict[num] = check_params
+
+    return render_template('calibdq_tellie.html',run_numbers=run_dict.keys(),run_info=run_dict.values())
+
+@app.route('/calibdq_tellie/<run_number>/')
```

Comment:
If this is always supposed to be an int you can use the decorator `@app.route('/calibdq_tellie/<int:run_number>')` and it will automatically get cast to an int for you.